TMessagesProj/src/main/java/org/telegram/messenger/MessagesStorage.java

```diff
@@ -743,6 +750,28 @@ public void run() {
         });
     }
 
+    public boolean requestMeExist(HashMap<Long, Integer> pushDialogs) {
```

Comment:
add method for check there is at least one dialog with call for current user
check special for push notifications